collector/src/benchmark/mod.rs

```diff
@@ -205,7 +205,7 @@ impl Benchmark {
         let iterations = iterations.unwrap_or(self.config.runs);
 
         let profiles: Vec<Profile> = profiles
-            .into_iter()
+            .iter()
             .copied()
```

Comment:
It seems strange to have `into_iter` and `copied` used together. Clippy suggested changing `into_iter` to `iter`, but maybe you could instead keep `into_iter` and remove the `copied`?

Reply:
Or maybe this is https://rust-lang.github.io/rust-clippy/master/index.html#/into_iter_on_ref ?

Reply:
Or maybe this is https://rust-lang.github.io/rust-clippy/master/index.html#/into_iter_on_ref ?

Reply:
Yeah it's `into_iter_on_ref`.